secure_module/zephyr/spm/src/main.c

```diff
@@ -62,9 +62,31 @@
 blst_scalar sk;
 blst_scalar secret_keys_store[10];
 blst_scalar sk_sign;
+char public_keys_hex_store[960];
+int keys_counter = 0;
 
 __TZ_NONSECURE_ENTRY_FUNC
-int public_key_to_sk(char * public_key_hex, char* public_keys_hex_store, int keys_counter, int offset){
+int get_keys_number(){
```

Comment:
a better name would be get_keystore_size

Reply:
Done